src/View/Helper/UrlHelper.php

```diff
@@ -41,6 +41,62 @@ public function build($url = null, $full = false)
     }
 
     /**
+     * Generate URL for given image file. Depending on options passed provides full URL
```

Comment:
I would do:

Short desc: Generate URL for given image file

Rest long desc.

Same goes for the rest of the proxies.